Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/PropertyEditorNodes.cpp

```diff
@@ -96,6 +96,8 @@ namespace AZ::DocumentPropertyEditor::Nodes
         system->RegisterNodeAttribute<ContainerActionButton>(ContainerActionButton::Action);
         system->RegisterNodeAttribute<ContainerActionButton>(ContainerActionButton::OnActivate);
 
+        system->RegisterPropertyEditor<PrefabOverrideProperty>();
```

Comment:
+1 to this, let's keep the main generic code modules clean and... well, generic.

Reply:
Yeah I think this is hitting a growing pain of "how do non-RPE systems register to the DPE?" - there should probably be a bus or something that single-shot notifies consumers when `PropertyEditorSystemInterface` is available so that other systems (inc gems) can plug in. @alexmontAmazon / @srikappa-amzn feel free to ping me on that if you've got design questions

Reply:
While refactoring this code, I realized that even if I don't call RegisterPropertyEditor<PrefabOverrideProperty>() , it still works and I can see the override icon. So I removed this registering part altogether.

@NicholasVanSickle , what exactly does RegisterPropertyEditor<>() do and how does it work even without it? Can we skip this for now?